### ISSUE TITLE
Yarn PR's do not include offline mirror files

### ISSUE THREAD

**Issue by phawxby** [dependabot/dependabot-core#2104]
If yarn is configured to use [offline mirror](https://yarnpkg.com/blog/2016/11/24/offline-mirror/) then during an install/update the tarballs are stored in a directory which is committed to the repo. This means the repo still works if flying, a package goes missing from npm, etc. It also happens to speed up the install process quite significantly. 

The PR's created by dependabot do not include the updated/removed tarballs.

**Comment by Austio** [comment]
@infin8x Any updates on this concept?  Looks like this works with go/ruby but yarn offline mirror is missing (also may be easy change due to already respecting .yarnrc file.

**Comment by infin8x** [comment]
Unfortunately I'm not on the Dependabot team anymore. @mayakacz or @feelepxyz may be able to get you in the right direction.

**Comment by feelepxyz** [comment]
@Austio we haven't had a chance to look at this but we have a few issues that are related to the fact that we only partially insert the `.yarnrc` into updates (dependabot only extracts the registry if it's set to registry.npmjs.org). We also don't clone the full repo contents for javascript projects so would have to do that before updating the vendored dependencies.

We're currently focusing on update existing ecosystems so will be a while before we can look at this.

**Comment by jeffwidman** [comment]
Duplicate of:
* https://github.com/dependabot/dependabot-core/issues/462
